migrate/20240731_firewall_location_not_null.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:firewall) do
+      set_column_not_null :location
```

Comment:
Should we shift this migration to the latest commit in this PR? Even though you've updated the existing firewalls, new ones can still be created until all commits are deployed.

Reply:
Good point! I did it.